layouts/partials/kubectl-reference-index.html

```diff
@@ -0,0 +1,48 @@
+        <h3 class="font-weight-bold mt-4 mb-1" id="{{ $sectionId }}"><a href=" {{ .RelPermalink }}">{{- $childPage.Title -}}</a></h3>
+
+        <!-- Details section for each section page -->
+        <p class="content-{{ $sectionId }}">
+            <details class="details-{{ $sectionId }}">
+                <summary class="card-header text-monospace summary-{{ $sectionId }}">{{- $childPage.Title -}}</summary>
+                <div class="border-bottom description-{{ $sectionId }}">
+                    {{- $content := $childPage.Content -}} <!-- Get child page content -->
+                    {{- $content = replaceRE "<h2 id=\"see-also\">.*?<\\/h2>\\s*<ul>([\\s\\S]*?)<\\/ul>" "" $content -}} <!-- Remove the 'See Also' section and its content -->
+                    {{- $content = replaceRE "<h2.*?>(.*?)<\\/h2>" "<h2>$1</h2>" $content -}} <!-- Remove 'id' attribute to eliminate the redundant deep link for sub-sections -->
+                    {{- $content = replaceRE "<table(\\s+[^>]*)?>" "<table class='table table-striped table-responsive'$1>" $content -}} <!-- Add 'class' attribute to table for similar look and feel to other tables on site -->
+                    {{- $content | safeHTML -}} <!-- Display modified content -->
+                </div>
+            </details>
+
+            <!-- Nested details section for sub-command child pages -->
+            {{- range $subPage := $childPage.RegularPages -}}
+                {{- $subSectionId := (replace $subPage.Title " " "-") | lower -}} <!-- Generate unique ID for each sub-command page -->
+                <details class="details-{{ $subSectionId }}">
+                    <summary class="card-header text-monospace summary-{{ $subSectionId }}">{{- $subPage.Title -}}</summary>
+                    <div class="border-bottom description-{{ $subSectionId }}">
+                        {{- $subPageContent := $subPage.Content -}} <!-- Get sub-section page content -->
+                        {{- $subPageContent = replaceRE "<h2 id=\"see-also\">.*?<\\/h2>\\s*<ul>([\\s\\S]*?)<\\/ul>" "" $subPageContent -}} <!-- Remove the 'See Also' section and its content -->
+                        {{- $subPageContent = replaceRE "<h2.*?>(.*?)<\\/h2>" "<h2>$1</h2>" $subPageContent -}} <!-- Remove 'id' attribute to eliminate the redundant deep link for sub-sections -->
+                        {{- $subPageContent = replaceRE "<table(\\s+[^>]*)?>" "<table class='table table-striped table-responsive'$1>" $subPageContent -}} <!-- Add 'class' attribute to table for similar look and feel to other tables on site -->
+                        {{- $subPageContent | safeHTML -}} <!-- Display modified content -->
+                    </div>
+                </details>
+            {{- end -}}
+        </p>
+    {{- end -}}
+</div>
+
+<!-- Display auto-generated information block -->
+{{- partial "docs/auto-generated-pageinfo.html" . -}}
```

Comment:
This feels a bit like cheating, although I understand the intent. It's fragile though, and doesn't update the right-hand navigation.

Reply:
There is potential to make the necessary changes to update the right-hand menu, although it will entail modifying a few default layout files. However, integrating this directly into the generator code would be a smoother solution overall.